### DISCUSSION TITLE
The name of PersistMiddleware in PersistOptions is overrided by the name of store

### DISCUSSION THREAD

**Discussion by GoodbyeNJN** [udecode/zustand-x#46] [Q&A] [answered]
I found related code at this line:
https://github.com/udecode/zustood/blob/main/packages/zustood/src/createStore.ts#L49

I'm not sure if this feature is intentional, but I suppose it should be supported that the name of PersistMiddleware is different from the name of store.

If you agree it, I will create a PR.

**Comment by zbeyens** [comment]
Sure, please open the PR :)

**Comment by GoodbyeNJN** [comment, marked as answer]
PR https://github.com/udecode/zustood/pull/47 has been merged.
